IMPLEMENTATION_COVERAGE.md

```diff
@@ -1019,7 +1019,6 @@
 - [ ] bundle_instance
 - [ ] cancel_bundle_task
 - [ ] cancel_capacity_reservation
-- [ ] cancel_capacity_reservation_fleets
```

Comment:
@bblommers Is `make implementation_coverage` expected to remove these commands? I ran it in order to update the `ses` module coverage, but it looks like it goes through the entire repo and updates all the modules. I'm worried it's accidentally removing commands that are supposed to be kept. If that's the case, I can revert this commit and manually update the `ses` section.

Reply:
It's not uncommon, to see methods removed. The script uses botocore in the background to check which methods AWS supports, and every now and then the AWS team will remove methods or an entire service completely. So nothing to worry about.